src/main/java/teammates/common/datatransfer/attributes/FeedbackQuestionAttributes.java

```diff
@@ -31,6 +31,7 @@
     public List<FeedbackParticipantType> showResponsesTo;
     public List<FeedbackParticipantType> showGiverNameTo;
     public List<FeedbackParticipantType> showRecipientNameTo;
+    public boolean allowRichText;
```

Comment:
Shouldn't this be specific to TEXT question?

Reply:
Ok. Will localize it to text question only.

Reply:
I hope you notice that it does not make sense for other question types (at least rank/rubric etc.)

Reply:
Yes. It does disturb me a little when I fill in that field for other question type due to typing. Should it be a sidecar attribute for TEXT question alone instead of one baked into the question?

Reply:
> Regarding your concern, should I really split it in the Backend and handle it separately?

What do you mean by handle it separately?

Frankly speaking, you have to modify `FeedbackTextQuestionDetails` (i.e. the backend) to get things to work.

Reply:
Yup. I realised and refactored it...
It should now be contained within FeedbackTextQuestionDetails